rfcs/text/0000-rfc-process-updates.md

```diff
@@ -0,0 +1,149 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# RFC Name
+
+*Replace "RFC Name" above with the name of your RFC. Keep it short and descriptive.*
+
+## Summary
+
+The Data Coordination Platform (DCP) established the Request For Comments (RFC) process on 
+[September 17th 2018](https://github.com/HumanCellAtlas/dcp-community/pull/26) in order to address the gap of having a
+coordinated, standardized, and repeatable process by which decisions could be made. Since this time, the DCP has evolved
+and likewise the RFC process is in need of some fine tuning. This RFC proposes some changes in order to reflect the new
+DCP community's needs.  
+
+## Author(s)
+
+*Recommended format for Authors:*
+
+ [Arathi Mani](mailto:arathi.mani@chanzuckerberg.com)
+ 
+ [Dave Rogers](mailto:dave@clevercanary.com)
+
+## Shepherd
+***Leave this blank.** This role is assigned by DCP PM to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## Motivation
+
+Over the last year since the original RFC process was accepted, the DCP project has grown both in size and complexity.
+As of this writing, 11 RFCs have been accepted in addition to many charters. Not so unexpectedly, there have been pain
```

Comment:
Not sure what charters has to do with the RFC process.

Reply:
I thought Charters also follow the RFC process?

Reply:
Charters were _bootstrapped_ before the RFC process - https://github.com/HumanCellAtlas/dcp-community/tree/master/charters - so they have their own process.